src/odemis/gui/cont/tabs.py

```diff
@@ -6405,12 +6405,12 @@ def add_axis(axisname, comp, label=None):
 
             # if the spec_switch mirror is not positioned either on ACTIVE or
             # DEACTIVE position move it to the default (DEACTIVE) position
-            if not almost_equal(main.spec_switch.position.value["x"],
-                                spec_switch_data[model.MD_FAV_POS_DEACTIVE]["x"]):
-                if not almost_equal(main.spec_switch.position.value["x"],
-                                    spec_switch_data[model.MD_FAV_POS_ACTIVE]["x"]):
-                    self._spec_switch_f = main.spec_switch.moveAbs(spec_switch_data[model.MD_FAV_POS_DEACTIVE])
-                    self._spec_switch_f.add_done_callback(self._on_specswitch_button_done)
+            if ((not almost_equal(main.spec_switch.position.value["x"],
+                                  spec_switch_data[model.MD_FAV_POS_DEACTIVE]["x"]))
+                    and (not almost_equal(main.spec_switch.position.value["x"],
```

Comment:
Can you adjust the indentation to align the `and` with the *second* parenthesis after the `if`.

Reply:
Yeah I agree this if statement became really cluttered and even PEP8 is losing it there.

Reply:
If it is this hard to properly align an if statement it is a sign that it is probably also unreadable. For readability you could also choose to clean it up to something like this:
```python
    pos_x = main.spec_switch.position.value["x"]
    pos_deactive = spec_switch_data[model.MD_FAV_POS_DEACTIVE]["x"]
    pos_active = spec_switch_data[model.MD_FAV_POS_ACTIVE]["x"]
    if ((not almost_equal(pos_x, pos_deactive))
            and (not almost_equal(pos_x, pos_active))):
```

Reply:
Yes! I just did exactly that already, thanks for pointing that out